huawei_lte_api/Connection.py

```diff
@@ -5,7 +5,7 @@
 
 import requests
 
-from huawei_lte_api.Session import Session
+from huawei_lte_api.Session import Session, _LOGGER
```

Comment:
Hmm, `logging.getLogger(__name__)`  should be created in each file separately 
so pls remove the import and put `_LOGGER = logging.getLogger(__name__)` after import block